python/Ganga/GPIDev/Adapters/IGangaFile.py

```diff
@@ -71,6 +72,15 @@ def put(self):
         """
         raise NotImplementedError
 
+    def copyTo(self, targetPath):
+        """
+        Copy a the file to the local storage using the get mechanism
+        Args:
+            targetPath (str): Target path where the file is to copied to
+        """
+        self.localDir = targetPath
```

Comment:
Tbh I wish I knew where the `get()` method puts the file every time you call it but it changes too much. This is what was being used in the past in place of a copy method. Already in ganga in code dating back a _long_ time. Is there a better/safer way to copy a file to a required destination.